src/shell_install/nushell.rs

```diff
@@ -113,7 +113,7 @@ def --env br [
     let cmd_file = ([ $nu.temp-path, $"broot-(random chars).tmp" ] | path join)
     touch $cmd_file
     if ($file == null) {
-        ^broot --outcmd $cmd_file $args
+        ^broot --outcmd $cmd_file ...$args
     } else {
         ^broot --outcmd $cmd_file $args $file
```

Comment:
shouldn't we do the same in this line?

Reply:
Absolutely, I didn't try `br` with a command, and it logically also throws the error:
```
Error:   × Automatically spreading lists is deprecated
    ╭─[/home/ludo/.config/local/share/broot/launcher/nushell/5:92:1]
 92 │     } else {
 93 │         ^broot --outcmd $cmd_file $args $file
    ·                                   ──┬──
    ·                                     ╰── Spreading lists automatically when calling external commands is deprecated and will be removed in 0.91.
 94 │     }
    ╰────
  help: Use the spread operator (put a '...' before the argument)
```

Reply:
I took the time to test it in every condition I can think of: without argument and with every argument `br` can have, and didn't have anymore error.
Do you think on a specific use case we should test, or should we give the go?

Reply:
I think we are good to go